flink-table/flink-table-api-java/src/main/java/org/apache/flink/table/operations/OperationUtils.java

```diff
@@ -43,9 +43,7 @@ public class OperationUtils {
      * @return string with increased indentation
      */
     static String indent(String item) {
-        return "\n"
-                + OPERATION_INDENT
-                + item.replace("\n" + OPERATION_INDENT, "\n" + OPERATION_INDENT + OPERATION_INDENT);
+        return "\n" + OPERATION_INDENT + item.replaceAll("\n", "\n" + OPERATION_INDENT);
```

Comment:
Is it guaranteed that the `item.replaceAll` has no impact on string literals? Maybe we should use SQL escaping in literals to avoid this?

Reply:
I changed the `OperationUtils#indent` in a way it should leave escaped `\n` untouched.